src/SupervisionSuccess/components/Chart/Chart.js

```diff
@@ -141,8 +140,8 @@ const Chart = ({ isError, data, startYear, isNotAvailable2020 }) => {
               padding: 20,
               fontSize: isMobile ? 12 : 16,
               precision: 0,
-              suggestedMax: min === max ? max + Y_AXIS_OFFSET : max + (max - min) * VERTICAL_OFFSET,
-              suggestedMin: min === max ? max - Y_AXIS_OFFSET : min - (max - min) * VERTICAL_OFFSET,
+              suggestedMax: maxY + (maxY - minY) * VERTICAL_OFFSET,
```

Comment:
Follow-up to my comment below, if `min` is higher than `max` (e.g. max=118 and min=120 as in your test below), then won't this produce a `suggestedMax` that is lower than the `suggestedMin`? What does Chart.js do in response to that? Something feels off.

Reply:
Added fix for that issue.